hw/dv/sv/dv_utils/dv_macros.svh

```
@@ -71,6 +71,14 @@
     _inst_name_ = _type_name_::type_id::create(`"_inst_name_`", this);
 `endif
 
+`ifndef DUT_HIER
+  `define DUT_HIER tb.dut
+`endif
+
```

We define `dut_instance` in our HJson config. Lets pass this macro by default in our common sim cfg, rather than set a default here. We should fail immediately this macro is not set externally, rather than go with a likely bogus path and it manifesting as a totally unrelated issue that is harder to debug.